lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3795,7 +3795,10 @@ LogicalResult StmtEmitter::visitStmt(InstanceOp op) {
     // lowered to wire.
     OutputOp output;
     if (!elt.isOutput()) {
-      emitExpression(portVal, ops, LowestPrecedence);
+      if (isZeroWidth && isa_and_nonnull<ConstantOp>(portVal.getDefiningOp()))
+        os << "/* Zero width */";
+      else
```

Comment:
Probably we don't have to restrict the op to be ConstantOp. 
nit: please use `emitZeroWidthIndexingValue` for consistency.
```suggestion
      if (isZeroWidth)
        emitZeroWidthIndexingValue(os);
      else
```

Reply:
If I don't restrict to constant op, a few cases get omitted. like results from other instances and module arguments. based on the tests, they appear useful and intentional. is this functionality we're willing to lose for implementation generality?

Reply:
Thanks, that makes sense to me. It sounds better :)